stdlib/LinearAlgebra/test/symmetric.jl

```diff
@@ -571,6 +571,20 @@ end
     end
 end
 
+@testset "eigendecomposition with lapack_method" begin
+    for T in (Float64, ComplexF64)
+        n = 4
+        A = T <: Real ? Symmetric(randn(T, n,n)) : Hermitian(randn(T, n,n))
+        d, v = eigen(A)
+        for lapack_method in (:syev, :syevr, :syevd)
+            @test eigvals(A, lapack_method) ≈ d
+            d2, v2 = eigen(A, lapack_method)
```

Comment:
First of all, I'd suggest to test for type stability, and then add a type assertion above if necessary.

```suggestion
            @test (@inferred eigvals(A, lapack_method)) ≈ d
            d2, v2 = @inferred eigen(A, lapack_method)
```

Reply:
Thanks, I added them.